### PR TITLE
docs: Update Developer Guide

### LINKED ISSUE #7094: update developer guide

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the developer guide at the connect to the debug console section.

Fixes #7094

### CODE REVIEW

File: docs/Developer-Guide.md [hunk 1]
```diff
@@ -587,10 +587,15 @@ $ sudo kata-monitor
 
 #### Connect to debug console
 
-Command `kata-runtime exec` is used to connect to the debug console.
+You need to start a container for example:
+```bash
+$ sudo ctr run  --runtime io.containerd.kata.v2 -d docker.io/library/ubuntu:latest testdebug
+```
+
+Then, you can use the command `kata-runtime exec` to connect to the debug console.
```
Comment (stevenhorsman) [inline docs/Developer-Guide.md:595]: I wonder if adding that the sandbox id is the parameter needed would help with clarity here?
```suggestion
Then, you can use the command `kata-runtime exec <sandbox id>` to connect to the debug console.
```

**Review by stevenhorsman — APPROVED** [review]
Thanks @GabyCT - I think this helps make it clearer - I just have a minor suggestion that might also be worth considering?

File: docs/Developer-Guide.md (hunk 1 above)
Comment (GabyCT) [reply docs/Developer-Guide.md]: @stevenhorsman thanks for the feedback, changes applied

**Review by dborquez — APPROVED** [review]
lgtm

**Pull request merged**